sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/DeviceInfoHelper.java

```diff
@@ -147,12 +147,23 @@ private static String getScreenSize(Context context) {
         /* Guess resolution based on the natural device orientation */
         int screenWidth;
         int screenHeight;
-
-        //noinspection ConstantConditions
-        Display defaultDisplay = ((WindowManager) context.getSystemService(Context.WINDOW_SERVICE))
-                .getDefaultDisplay();
+        Display defaultDisplay;
         Point size = new Point();
-        defaultDisplay.getSize(size);
+
+        /* Use DeviceManager to avoid android.os.strictmode.IncorrectContextUseViolation when StrictMode is enabled on API 30. */
```

Comment:
Comments says use DeviceManager but seems there is not DeviceManager

Reply:
It's a typo, if you wish you can open a contribution PR with a fix, or I can do it later.